get_branch_rank.py

```diff
@@ -0,0 +1,122 @@
+import requests
+from bs4 import BeautifulSoup
```

Comment:
@Demfier Please add some script documentation at the top of the file. This should include a simple way to use this script, the pre-requisuites that are required, and how to install them (link to a gist with a `pip` `requirements.txt` file would be really appreciated).